web/components/view/ShowHideTextView/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import arrow from '../../../assets/arrow.svg';
+
+class ShowHideTextView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { showDropDown: false };
+    this.handleClick = () => {
```

Comment:
I was thinking it would be a method of the class (like just declaring `handleClick = () => { ... }` in line 14)

Reply:
Ah okay, I can change that